src/common/http/http.cc

```diff
@@ -349,6 +350,71 @@ response_t http_impl::Post(
   }
 }
 
+response_t http_impl::Post(
+        const authservice::config::common::Endpoint &endpoint,
+        const std::map<absl::string_view, absl::string_view> &headers,
+        absl::string_view body,
+        boost::asio::io_context& ioc,
+        boost::asio::yield_context yield) const {
+  spdlog::trace("{}", __func__);
+  try {
+    int version = 11;
+
+    ssl::context ctx(ssl::context::tlsv12_client);
+    // TODO: verify_peer should be used but is not currently working.
```

Comment:
Is this still true?

Reply:
I don't know, I didn't really touch any of the logic in here, what was the problem you were originally having with it?

Reply:
Previously using `verify_peer` instead of `verify_none` would result in an error which I cannot remember. We need to make sure it works now though else we have a massive security issue.

Reply:
Will try this out and see if I can fix it in a separate pull request after this one has been merged.